src/start_menu/main_settings/main_settings.cpp

```diff
@@ -0,0 +1,61 @@
+#include "weight.h"
+#include "speed.h"
+#include "size.h"
+#include "main_settings.h"
```

Comment:
The `include` of this files header should always be on top. The rest of them should follow alphabetically.